src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -579,6 +579,11 @@ public static void Delete(String path, bool recursive)
             String fullPath = Path.GetFullPath(path);
             FileSystem.Current.RemoveDirectory(fullPath, recursive);
         }
+
+        public static string[] GetLogicalDrives()
```

Comment:
I'd also like to see the existing less-than-perfect bits drop off of FileSystem long-term. If we expose an IFileSystem or FileSystem itself (which I ultimately want to so people can address testability issues with their own code) I would want it to only include things that people can have confidence in when coding cross platform without having to conditionalize their code or understand the difference.

I'll also note that it feels "icky". It is already abstracted in DriveInfoInternal and the current FileSystem has no implementation. I can either add an implementation in FileSystem or derive in every class to have them call the exact same line of code. Not a justification, but another reason I'm feeling hesitant.